index/scorch/scorch.go

```diff
@@ -582,7 +600,9 @@ func (s *Scorch) StatsMap() map[string]interface{} {
 	m["index_time"] = m["TotIndexTime"]
 	m["term_searchers_started"] = m["TotTermSearchersStarted"]
 	m["term_searchers_finished"] = m["TotTermSearchersFinished"]
+	m["num_bytes_read_query_time"] = m["TotBytesReadQueryTime"]
 	m["num_plain_text_bytes_indexed"] = m["TotIndexedPlainTextBytes"]
+	m["num_analysis_bytes_indexed"] = m["TotIndexedAnalysisBytes"]
```

Comment:
It's good to have both the stats share the same prefix like "num_bytes_".
So, please reword the stats..